app/src/app/search/search-bar/search-bar.component.ts

```diff
@@ -148,6 +138,9 @@ export class SearchBarComponent implements OnInit, OnDestroy {
     switch (searchForm) {
       case 'text':
         tag.searchText = searchTerm.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&');
+        if (!isNaN(Number(searchTerm))) {
```

Comment:
I would extend this check by checking for integers. At the moment, float values are also allowed.

For Example like: 
 if (!isNaN(searchTerm) && Number.isInteger(searchTerm)) {